vendor/archive/tar/example_test.go

```diff
@@ -1,80 +0,0 @@
-// Copyright 2013 The Go Authors. All rights reserved.
-// Use of this source code is governed by a BSD-style
```

Comment:
Wonder how these got in (through https://github.com/moby/moby/commit/7acea2a243d25c061d12a2a2f8bbd4e5955a85f4) without the vendor check failing

Reply:
Looks like the `vendor` check never ran on that PR. It has run on this one, so it's not completely broken.